app/assets/stylesheets/components/_robots.scss

```diff
@@ -0,0 +1,14 @@
+.card {
```

Comment:
remove this file and modify the card classes in _cards.scss

Reply:
Resolved